erpnext/controllers/accounts_controller.py

```diff
@@ -383,13 +383,14 @@ def on_trash(self):
 					== 1
 				)
 			).run()
-			frappe.db.sql(
-				"delete from `tabGL Entry` where voucher_type=%s and voucher_no=%s", (self.doctype, self.name)
-			)
-			frappe.db.sql(
-				"delete from `tabStock Ledger Entry` where voucher_type=%s and voucher_no=%s",
-				(self.doctype, self.name),
-			)
+			gle = frappe.qb.DocType("GL Entry")
```

Comment:
These sql -> qb refactoring could've been a separate PR. Always better to keep the context of your change isolated to specific issue or feature

Reply:
Okay sure, I thought the GitHub checks would fail if I pushed it as a sql query, so I refactored it.